main.c

```diff
@@ -935,6 +935,7 @@ int main(int argc, char **argv)
 	g_main_loop_unref(loop);
 	tcmulib_close(tcmulib_context);
 	tcmu_config_destroy(tcmu_cfg);
+	darray_free(handlers);
 
```

Comment:
This was just fixed in commit, so will drop this patch.

commit aaf121b7441a939b1f60fcddeacad7c69f0c2b1e
Author: Xiubo Li <lixiubo@cmss.chinamobile.com>
Date:   Thu Jul 20 10:11:41 2017 +0800

    tcmur: free useless darray type handlers memory.